doc/example/patterns_async_call.cpp

```diff
@@ -25,14 +26,23 @@ namespace
 }
 //]
 
+namespace mock_test
+{
+    my_class::my_class( base_class& b): b(b){}
+    void my_class::flush()
+    {
+        static int secret_value = 7;
```

Comment:
Why the secret? 😀 Can’t it just be `count` or something like that?

Reply:
Good point